src/main/java/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.java

```diff
@@ -2625,7 +2625,30 @@ public final class JavadocTokenTypes {
     /** Basefont html tag. */
     public static final int BASEFONT_TAG = JavadocParser.RULE_basefontTag + RULE_TYPES_OFFSET;
 
-    /** Br html tag. */
+    /**
+     * Br html tag.
+     *
+     * <p><b>Example:</b></p>
+     * <pre>{@code <br> line breaks<br>}</pre>
```

Comment:
Make second br to be `<br/>`

https://dev.to/scalertopics/difference-between-and-tag-in-html-5350#:~:text=br%20tag%20is%20used%20to,%2Fbr)%20while%20using%20br.

Just to show diversity and how both are supported